Docker-Swarm-deployment/single-node/immudb/example-secrets/secrets/config.json

```diff
@@ -1,59 +1,47 @@
-{   "immudb_host": "immudb", 
-    "immudb_default_user": "immudb",
-    "immudb_default_user_password": "immudb",
-    "change_admin_password": true,
-    "admin_password": "/run/secrets/password/admin-password",
-      
-       
-    "database": [
-
-        {
-            "database_name": "iudxauth",
-            "table": "table_auditing(id VARCHAR[128] NOT NULL, body VARCHAR NOT NULL,userid VARCHAR[128] NOT NULL,endpoint VARCHAR[128] NOT NULL,method VARCHAR[32] NOT NULL,time INTEGER NOT NULL,PRIMARY KEY id)",
-            "indexing_on": "table_auditing(userid, endpoint, time)",
-            "flush_threshold": 1000,
-            "sync_threshold": 1000,
-            "cleanup_percentage": 0.1
-
-          },
-          {
-            "database_name": "iudxcat",
-            "table": "auditingtable(id VARCHAR[128] NOT NULL, userRole VARCHAR[64] NOT NULL,userID VARCHAR[128] NOT NULL,iid VARCHAR[250] NOT NULL,api VARCHAR[128] NOT NULL,method VARCHAR[32] NOT NULL,time INTEGER NOT NULL,iudxID VARCHAR[256] NOT NULL,PRIMARY KEY id)",
-            "indexing_on": "auditingtable(userID, iudxID, time, api)",
-            "flush_threshold": 600,
-            "sync_threshold": 600,
-            "cleanup_percentage": 0.1
-          },
-          {
-            "database_name": "iudxrsorg",
-            "table": "rsaudit(id VARCHAR[128] NOT NULL,api VARCHAR[128] NOT NULL,userid VARCHAR[128] NOT NULL,epochtime INTEGER NOT NULL,resourceid VARCHAR[256] NOT NULL,isotime VARCHAR[64] NOT NULL,providerid VARCHAR[128] NOT NULL,size INTEGER, PRIMARY KEY id)",
-            "indexing_on": "rsaudit(userid, epochtime, providerid, resourceid)",
-            "flush_threshold": 1000,
-            "sync_threshold": 1000,
-            "cleanup_percentage": 0.1
-          }
-    ],
-
-    "users": [
-
-        {
-            "username": "iudx_auth",
-            "password": "/run/secrets/password/auth-password",
-            "database_name": "iudxauth"
-          },
-          {
-            "username": "iudx_cat",
-            "password": "/run/secrets/password/cat-password",
-            "database_name": "iudxcat"
-          },
-          {
-            "username": "iudx_rs",
-            "password": "/run/secrets/password/rs-password",
-            "database_name": "iudxrsorg"
-        }
-    ]
-
-
-
+{
+  "immudb_host": "immudb",
+  "immudb_default_user": "immudb",
+  "immudb_default_user_password": "immudb",
+  "change_admin_password": true,
+  "admin_password": "/run/secrets/password/admin-password",
+  
+  
+  "database": "iudx",
+  "tables": [
+  {
+    "table": "auditing_acl_apd( id VARCHAR[128] NOT NULL, userid VARCHAR[128] NOT NULL, endpoint VARCHAR[128] NOT NULL, method VARCHAR[128] NOT NULL, body VARCHAR[128] NOT NULL,size INTEGER NOT NULL,isotime VARCHAR[128] NOT NULL, epochtime INTEGER NOT NULL, PRIMARY KEY id)",
+    "indexing_on": "auditing_acl_apd(userid,endpoint,epochtime)",
+    "flush_threshold": 1000,
```

Comment:
flush threshold, sync_threshold, cleanup_percentage is database settings. can remove from table json object and put  something like this :

```
  "database": "iudx",
  "flush_threshold": 600,
  "sync_threshold": 600,
  "cleanup_percentage": 0.1
 "tables": [
  {
    "table": "auditing_acl_apd( id VARCHAR[128] NOT NULL, userid VARCHAR[128] NOT NULL, endpoint VARCHAR[128] NOT NULL, method VARCHAR[128] NOT NULL, body VARCHAR[128] NOT NULL,size INTEGER NOT NULL,isotime VARCHAR[128] NOT NULL, epochtime INTEGER NOT NULL, PRIMARY KEY id)",
    "indexing_on": "auditing_acl_apd(userid,endpoint,epochtime)"
    },
  {
    "table": "auditingtable(id VARCHAR[128] NOT NULL, userRole VARCHAR[64] NOT NULL,userID VARCHAR[128] NOT NULL,iid VARCHAR[250] NOT NULL,api VARCHAR[128] NOT NULL,method VARCHAR[32] NOT NULL,time INTEGER NOT NULL,iudxID VARCHAR[256] NOT NULL,PRIMARY KEY id)",
    "indexing_on": "auditingtable(userID, iudxID, time, api)"
  }
  ]
```